### ISSUE TITLE
Calibration file blocks Octoprint-TFT from showing 

### ISSUE THREAD

**Issue by wlcina** [mcuadros/OctoPrint-TFT#74]
I successfully installed Octopi-TFT.
I am using waveshare TFT display with its driver
`git clone https://github.com/waveshare/LCD-show.git`
- After checking I found out display is not calibrated, so I executed calibration tool
```
cp LCD-show/xinput-calibrator_0.7.5-1_armhf.deb ~/
sudo dpkg -i -B xinput-calibrator_0.7.5-1_armhf.deb
```
- Running calibrator and finish calibration
`DISPLAY=:0.0 xinput_calibrator`
- I got some calibration data, I was supposed to save the calibration data to 99-clibration.conf file
```
sudo mkdir /etc/X11/xorg.conf.d
sudo nano /etc/X11/xorg.conf.d/99-calibration.conf - where I pasted them
```

```
Section "InputClass"
        Identifier      "calibration"
        MatchProduct    "ADS7846 Touchscreen"
        Option  "Calibration"   "259 3911 3757 156"
        Option  "SwapAxes"      "0"
EndSection
```
Then I reboot the device
`sudo reboot`
After that  the Octoprint-TFT wont show, just black screen.
If I remove calibration data file 99-calibration.conf and reboot the device, Octoprint-TFT is again on the screen, but I cant use it, as the display is not calibrated.

Anyone know what I am supposed to do?

**Comment by synman** [comment]
I think you are missing the actual display driver reference.  My Pi is not physically available to me right now but I have the exact same display on one of my Pi's and have it working fine.

Your x11 startup log likely states that you have no configured displays.

**Comment by synman** [comment]
here you go:

```
Section "InputClass"
        Identifier      "calibration"
        MatchProduct    "ADS7846 Touchscreen"
        Option  "Calibration"   "3932 300 294 3801"
        Option  "SwapAxes"      "1"
EndSection

Section "Device"
  Identifier "touchscreen"
  Driver "fbdev"
  Option "fbdev" "/dev/fb1"
EndSection
```

**Comment by synman** [comment]
additionally, mine is here:

./usr/share/X11/xorg.conf.d/99-calibration.conf

X11 will look in a couple different places so make sure you don't have a collision occurring.

**Comment by synman** [comment]
my apologies for the spamming... it looks like I just made sure I had the same file in both locations:

```
pi@octopi-laser:/etc/X11/xorg.conf.d $ cat 99-calibration.conf 
Section "InputClass"
        Identifier      "calibration"
        MatchProduct    "ADS7846 Touchscreen"
        Option  "Calibration"   "3932 300 294 3801"
        Option  "SwapAxes"      "1"
EndSection

Section "Device" 
  Identifier "touchscreen"
  Driver "fbdev"
  Option "fbdev" "/dev/fb1"
EndSection
```

**Comment by wlcina** [comment]
Thanks! You find a solution for me, what I did - deleted the calibration file, rebooted device. Then run the calibration to have calibration data. And finally created the file in both locations as you specified. (also added the "Device" section)

**Comment by synman** [comment]
Yeah, pretty sure it was the missing device section causing you problems.

Glad you got it working.  Congrats!
